public/app/features/search/page/components/FolderView.tsx

```diff
@@ -49,6 +49,8 @@ export const FolderView = ({
     const rsp = await getGrafanaSearcher().search({
       query: '*',
       kind: ['folder'],
+      sort: 'name_sort',
```

Comment:
this doesn't work because the alphabetical sort of the old search is differnetly named (alpha-asc), and the old search logic sorts in memory only if the `sort` is empty https://github.com/grafana/grafana/blob/c61b5e85b47defa15ad3c8df6ae5578739ae12f9/pkg/services/search/service.go#L85
-

Reply:
and we can't just use `alpha-asc` because it doesn't work well - it e.g. lists `Oncall` before `f1`, and `f1` before `oncall`